### DISCUSSION TITLE
Overide Duration?

### DISCUSSION THREAD

**Discussion by roblombardo** [robbinjanssen/home-assistant-ojmicroline-thermostat#156] [Q&A]
Great integration! I've integrated (2) UWG4 thermostats. When overriding the temperature, the override sets to "permanent". Is there a way to make the override temporary, similar to how the OJ Microline app does? Ideally I'd like the override to persist for only 2 hours and then fall back on schedule.

**Comment by robbinjanssen** [comment]
I'm not familiar with the UWG4, maybe @adamjernst can say more about this?

The OWD5 uses a mode called "Comfort" mode which sets the temp for x hours, is that what you are looking for?

**Reply by roblombardo** [reply]
I saw that in the configuration. I tried activating and setting it for 120 min., but the override was still set as "permanent".
